src/components/Dialog/Dialog.jsx

```diff
@@ -0,0 +1,68 @@
+import styles from './Dialog.module.css';
+import { createPortal } from 'react-dom';
+import { Box, Container, Grid, IconButton } from '@mui/material';
+import CssBaseline from '@mui/material/CssBaseline';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import CloseIcon from '@mui/icons-material/Close';
+import Button from '@mui/material/Button';
+import PropTypes from 'prop-types';
+
+const darkTheme = createTheme({
+  palette: {
+    mode: 'dark',
+  },
+});
```

Comment:
let's create `contstants `folder in `src` where we can create theme.js and reuse it in all app where it's needed
```suggestion
const darkTheme = createTheme({
  palette: {
    mode: 'dark',
  },
});
```